### PR TITLE
🐛 Source Google Sheets: migrate to advancedAuth

### LINKED ISSUE #26234: Source Google Sheets: Migrate to advancedAuth

### PR DESCRIPTION
**Pull request by tolik0**
## What
Migrate oAuth to advancedAuth.
#26234 

## How
Add advancedAuth to spec.json.

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-sheets/source_google_sheets/spec.yaml [hunk 1]
```diff
@@ -73,14 +73,46 @@ connectionSpecification:
...
+    complete_oauth_output_specification:
+      type: object
+      additionalProperties: false
+      properties:
+        refresh_token:
+          type: string
+          path_in_connector_config:
+            - credentials
+            - refresh_token
+        access_token:
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-google-sheets/source_google_sheets/spec.yaml:92]: I think we can skip it as it is not present in the spec

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-sheets/source_google_sheets/spec.yaml (hunk 1 above)
Comment (tolik0) [reply airbyte-integrations/connectors/source-google-sheets/source_google_sheets/spec.yaml]: Fixed

**Review by lazebnyi — APPROVED** [review]

**Pull request merged**